src/Sylius/Bundle/ShopBundle/templates/shared/order/show/summary/table/footer/shipping.html.twig

```diff
@@ -0,0 +1,3 @@
+<tr>
+    {% include '@SyliusShop/shared/order/table/shipping.html.twig' with {'order': hookable_metadata.context.order} %}
```

Comment:
It should be replaced by the twig hook with configured template in the YAML configuration